don't blindly overwrite $LD_LIBRARY_PATH (which breaks ld.gold linker because it requires a sufficiently recent libstdc++.so)
based on work by Kenneth Hoste (HPC-UGent)
diff -ruN sra-tools-2.11.0.orig/build/ld.linux.dlib.sh sra-tools-2.11.0/build/ld.linux.dlib.sh
--- sra-tools-2.11.0.orig/build/ld.linux.dlib.sh	2021-03-15 11:25:25.000000000 -0700
+++ sra-tools-2.11.0/build/ld.linux.dlib.sh	2021-06-10 16:54:23.070099000 -0700
@@ -83,8 +83,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
@@ -308,4 +307,4 @@
     echo "$MSG" >> $LOGFILE
 
     "$BUILD_DIR/scm-collect.sh" "$OBJS" "$SLIBS" | sort -u > "$TARG.md5"
-fi
\ No newline at end of file
+fi
diff -ruN sra-tools-2.11.0.orig/build/ld.linux.exe.sh sra-tools-2.11.0/build/ld.linux.exe.sh
--- sra-tools-2.11.0.orig/build/ld.linux.exe.sh	2021-03-15 11:25:25.000000000 -0700
+++ sra-tools-2.11.0/build/ld.linux.exe.sh	2021-06-10 16:54:37.368931000 -0700
@@ -96,8 +96,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
